model/model_training/custom_datasets/utils.py

```diff
@@ -1,4 +1,46 @@
-FILTER_BY_WORDS = ["openai"]
+# mostly taken from
+# https://huggingface.co/datasets/gozfarb/ShareGPT_Vicuna_unfiltered/blob/main/optional_clean.py,
+# https://huggingface.co/datasets/ehartford/WizardLM_alpaca_evol_instruct_70k_unfiltered/blob/main/wizardlm_clean.py
+FILTER_BY_WORDS = [
```

Comment:
This looks like duplicate code, can probably be shared with the `check_dataset_appearance` script

Reply:
used the code from utils.py